src/components/App/App.tsx

```diff
@@ -34,18 +34,33 @@ export const App: FunctionComponent = () => {
         break
       case '÷':
         if (rightOperand === 0) {
-          return false
+          newResult = 0
+          break
         }
-
         newResult /= rightOperand
+        break
     }
 
     setResult(newResult)
-    setDisplay(newResult.toString().toString().slice(0, 12))
-
+    
+    // Display "ERROR" when /0 
+    if (pendingOperator === "÷" && rightOperand === 0){
+      setDisplay("ERROR")
+    }
+    else{
+      setDisplay(newResult.toString().toString().slice(0, 12))
+    }
     return true
   }
 
+  // Handling display="ERROR"
+  const convertToNumberWithCheckIfNaN = (display: string) => {
+    if (Number.isNaN(Number(display))){
```

Comment:
Need a space before open curly bracket

```suggestion
    if (Number.isNaN(Number(display))) {
```